jax/_src/lax/control_flow/loops.py

```diff
@@ -1118,23 +1118,6 @@ def _scan_typecheck(bind_time, *in_atoms, reverse, length, num_consts,
       f'called with sequence whose items have type\n{_avals_short(x_avals_mapped)}')
   return [*init_avals, *y_avals], jaxpr.effects
 
-def _scan_pp_rule(eqn, context, settings):
```

Comment:
You can make a case for this (and other) pp rules being useful, but they have been disabled for a few years already.